tests/integration/solidity_contracts/conftest.py

```diff
@@ -174,9 +183,17 @@ async def _factory(bytecode: str, caller_eoa=None):
             starknet_contract_address,
             None,
         )
+
         await contract.write_bytecode(hex_string_to_bytes_array(bytecode)).execute(
             caller_address=kakarot.contract_address
         )
+
+        if storage is None:
```

Comment:
`storage is not None`

Reply:
Oops sorry about that, let me update this